utoipa-swagger-ui/Cargo.toml

```diff
@@ -12,15 +12,20 @@ authors = ["Juha Kukkonen <juha7kukkonen@gmail.com>"]
 rust-version.workspace = true
 
 [features]
+default = ["url"]
 debug = []
 debug-embed = ["rust-embed/debug-embed"]
+reqwest = ["dep:reqwest"]
+url = ["dep:url"]
```

Comment:
I don't see anything feature-flagged on url, so it looks like it's just always required?

Reply:
I think there are some places like [here](https://github.com/juhaku/utoipa/blob/403d716513612cb52bef0525640b9a96cd1bb18f/utoipa-swagger-ui/build.rs#L155).  

But in principle at least, url is necessary to parse `SWAGGER_UI_DOWNLOAD_URL` in all cases except in the case of vendored swagger_ui.zip so maybe the vendored flag is enough ?

Reply:
True, would be enough at this point, and would work without explicit `url` feature since the optional dependencies becomes implicit feature flags anyways. But it does not matter.